components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -70,7 +70,7 @@ export default class PostProfilePicture extends React.PureComponent {
         } = this.props;
         const isSystemMessage = PostUtils.isSystemMessage(post);
         const fromWebhook = PostUtils.isFromWebhook(post);
-        if (isSystemMessage && !compactDisplay && !fromWebhook) {
+        if (isSystemMessage && !compactDisplay && !fromWebhook && !user.is_bot) {
```

Comment:
Looks like `user` needs to check if not null since it's not required in this component.  Might also good to do the same as [post_header index](https://github.com/mattermost/mattermost-webapp/blob/e8485d29d6dfefd4900d2a7e71ab11d6af1a74a7/components/post_view/post_header/index.js#L14) where it passed `isBot` prop.

Reply:
Changed both to use the ternary operator.  Please let me know if this is what was requested.

Reply:
Do the latest changes match what you requested @saturninoabril ?

Reply:
Thanks, matched with the latest push.